Flagsmith.FlagsmithClient/Cache/FlagListCache.cs

```diff
@@ -18,7 +18,10 @@ protected FlagListCache(IDateTimeProvider dateTimeProvider, IFlags flags, int ca
         {
             _flags = flags;
             _dateTimeProvider = dateTimeProvider;
-            _timestamp = dateTimeProvider.Now();
+
+            //This is to ensure that the cache is stale on first run
+            _timestamp = dateTimeProvider.Now().Subtract(new TimeSpan(0, 0, cacheDurationInMinutes + 1, 0));
```

Comment:
This feels a bit hacky, can't we just allow `_timestamp` to be null and initialise it as such?

Reply:
I was trying to be quick. You are right, I can find something better than that. :)
Will fix and submit something today!